quickevent/app/quickevent/plugins/Event/src/services/oresultsclient.h

```diff
@@ -0,0 +1,53 @@
+#ifndef ORESULTSCLIENT_H
+#define ORESULTSCLIENT_H
+
+#pragma once
+
+#include "service.h"
+
+#include <QNetworkAccessManager>
+
+class QTimer;
+
+namespace Event {
+namespace services {
+
+class OResultsClientSettings : public ServiceSettings
+{
+	using Super = ServiceSettings;
+
+	QF_VARIANTMAP_FIELD(QString, a, setA, piKey)
+	QF_VARIANTMAP_FIELD2(int, e, setE, xportIntervalSec, 15)
+public:
+	OResultsClientSettings(const QVariantMap &o = QVariantMap()) : Super(o) {}
+};
+
+class OResultsClient : public Service
+{
+	Q_OBJECT
+
+	using Super = Service;
+public:
+	OResultsClient(QObject *parent);
+
+	//void run() override;
```

Comment:
IMO, `run()` & `stop()` must start/stop export timer, current service implementation cannot be stop, unless one set interval to zero.

Reply:
starting and stopping the service works fine.
However, i found out another bug. `Ok button` does not save settings, will fix shortly